rapid7_insightvm/plugin.spec.yaml

```diff
@@ -3723,7 +3820,28 @@ actions:
         type: boolean
         required: true
         example: True
-
+  asset_vulnerability_solution:
+    title: Get Asset Vulnerability Solutions
+    description: Returns the highest-superceding rollup solutions for a list of vulnerabilities on an asset
+    input:
+      asset_id:
+        title: Asset ID
+        description: The identifier of the asset
+        type: integer
```

Comment:
What is the highest possible value for this identifier? We need to find this out before we merge this - there is behavior in the product right now that will round integers over a certain number and will cause incorrect IDs. This should be a string to prevent that possibility.